contents/blog/how-our-community-works.md

```diff
@@ -0,0 +1,144 @@
+---
+date: 2024-01-30
+title: Why we built our own community platform
+rootPage: /blog
+category: PostHog news
+author: ["cory-watilo"]
+featuredImage: ../images/blog/how-our-community-works/og.png
+featuredImageType: full
+---
+
+import ImgForums from '../images/blog/how-our-community-works/forums.png'
+import ImgQuestion from '../images/blog/how-our-community-works/question.png'
+import ImgQuestionTraffic from '../images/blog/how-our-community-works/traffic.png'
+import ImgModeratorTools from '../images/blog/how-our-community-works/moderator.png'
+
+Hosted community platforms make it easy for startups to set up forums or a feature request board without pulling engineering resources away from building their core product.
+
+One popular platform you've likely seen before is [Discourse](https://www.discourse.org/). Given that it's feature-rich, it's obvious why many startups use it.
+
+But when we started building our community features as a relatively small startup, we took a different approach. We went this route to avoid the ghost town effect.
+
+## What is the ghost town effect?
+
+When creating a community from scratch, you don't instantly start out with a large, active user base. And if a prospective customer opens the forums only to see a bunch of categories with hardly any posts or activity, it can be a negative signal that the company lacks traction.
+
+This is specifically what we wanted to avoid.
+
+Some companies have been known to seed posts themselves in the beginning – to jump start the process. But we felt this approach is deceiving and dishonest. One of the guiding principles at PostHog is to always do the right thing by our customers. Our reputation is more important than that. One negative post on Hacker News for doing something sleazy like this can completely derail the trust we're trying to build.
+
+So our challenge: how do we jumpstart a community without resorting to cheap tactics?
+
+We had an idea…
+
+## Incorporating community into our docs
+
+To understand our approach, it's important to have some context as to our unique challenges a couple years ago.
+
+Our [technical docs](/docs) have existed from the start of the company. But with how quickly we've moved, many of our articles suffered from out-of-date information or a general lack of depth. We heard this from many customers on our ([now-retired](/blog/slack-closure)) community Slack, via our support channels, or on X.
+
+But helping customers in these various platforms doesn't solve the larger issue: if one person is asking, there's likely 10x more who have the same question but *haven't* reached out for a solution.
+
+Part of the reason we moved away from our community Slack is that rich information wasn't discoverable *outside* of Slack, and even that info would disappear into the abyss of the interwebs after a few months. (Slack hides older posts on free accounts, and their pricing doesn't scale for communities with thousands of users.)
```

Comment:
These first three sections don't totally work for me atm for a few reasons:

1. It's kind of like three intros leading into each other. 
2. It assumes a bit too much knowledge (you only mention the Slack in third section)
3. I don't think the ghost town effect needs this much explanation
4. I feel like avoiding the "ghost town effect" isn't a sufficiently compelling explanation for why we built our own. 

As a whole piece, this feels more like a dive into how we built the community with a token explanation for why did it this way tacked on at the start.

1. Put all the context in the intro... we recently closed our public Slack in favor of our website community. This was months in the making, but people keep asking us (i) how we built our own community (ii) why we didn't use an off-the-shelf thing like [insert example].

2. The next section should be about the problems with off-the-shelf options, or the benefits of building you own. You can talk about the ghost town effect here, but maybe also other factors: lack of control over look and feel, integration with the website etc.

3. Then jump into how we built it – Q&A was stage 1. This proved we could get traction, solved an obvious problem. Solved the ghost town effect. Stage 2 was building our forum.

4. The deep dive into the technical stuff

5. Maybe some details on what's coming up next – merging app and website auth etc.